.pylintrc

```diff
@@ -0,0 +1,292 @@
+[MASTER]
```

Comment:
It wasn't it was cribbed from another project.  I had no idea edx-pylint was a thing, but happy to use it if that makes sense.